osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPoint.cs

```diff
@@ -50,6 +50,7 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
+            Delay(-TransformStartTime); // So our follow points start their transforms from t=0 (the start of the map), rather from the lead in time (where the clock is right now).

```

Comment:
This is definitely not a hack. Follow points need their transforms to line up with the map. But if the state of the clock is anything but 0 (which is totally possible if we do a Seek before hand like in Player), then transforms will start too early.